.eslintrc.json

```diff
@@ -9,19 +9,19 @@
     "Atomics": "readonly",
     "SharedArrayBuffer": "readonly"
   },
-  "ignorePatterns": ["!.storybook"],
+  "ignorePatterns": ["!.storybook", "src/devtools/client/debugger/src/utils/editor/index.d.ts"],
   "parserOptions": {
     "ecmaFeatures": {
       "jsx": true
     },
     "ecmaVersion": 2018,
-    "project": ["tsconfig.json"],
+    "project": ["tsconfig.eslint.json"],
     "sourceType": "module"
   },
   "plugins": ["import", "react", "prettier", "@typescript-eslint"],
   "rules": {
-    "@typescript-eslint/no-base-to-string": "warn",
-    "@typescript-eslint/no-floating-promises": "error",
+    "@typescript-eslint/no-base-to-string": "off",
+    "@typescript-eslint/no-floating-promises": "off",
```

Comment:
> for instance, if you find the `might log as [Object object]` warnings important

That rule isn't there to guard against _logging_, but the generation of unique _keys_ (strings) used for our Suspense caches. (That actually caused a bug that I found and fixed recently– which led me to enable that rule.)

I think we need to leave it on.

Reply:
Maybe this rules really is too noisy and we'd be better with it off.

I think the `no-base-to-string` one is still worth keeping on though.

Reply:
Oh, if they are adding value I'm totally fine with going through and annotating the places that we don't care about them. I just noticed that there were a lot of violations that seemed harmless, so it seemed like we weren't really following the rule to begin with. I'll add some annotations and turn them back on.

Reply:
I think `no-base-to-string` adds value. I'm less concerned about the other one, given the number of places we seem to be okay with the "fire and forget" pattern.

I think the straw that broke the camel's back, so to speak, was noticing that all of our Suspense caches fire false negatives here– because they call async functions that never thrown (they always try/catch their inner promises and resolve/reject the "wakeable")